src/components/flexjar/senfase/SenFaseFlexjar.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from "react";
+import { Alert, Box, Button, Label, Radio, RadioGroup } from "@navikt/ds-react";
+import {
+  FlexjarFeedbackDTO,
+  useFlexjarFeedback,
+} from "@/data/flexjar/useFlexjarFeedback";
+import { ChevronDownIcon, ChevronUpIcon } from "@navikt/aksel-icons";
+import { defaultErrorTexts } from "@/api/errors";
+import AndreTilbakemeldingerTextArea from "@/components/flexjar/senfase/AndreTilbakemeldingerTextArea";
+import HvordanBrukerDuArenaTextArea from "@/components/flexjar/senfase/HvordanBrukerDuArenaTextArea";
+import { StoreKey, useLocalStorageState } from "@/hooks/useLocalStorageState";
+
+const texts = {
+  apneKnapp: "Vi ønsker å lære av deg",
+  anonym: "Anonym tilbakemelding",
+  send: "Send",
+  success: "Takk for din tilbakemelding!",
+  sporsmal:
+    "Trenger du fortsatt Arena i oppfølgingen av sykmeldte som nærmer seg maksdato?",
+};
+
+enum RadioOption {
+  JA = "Ja",
+  NEI = "Nei",
+}
+
+export default function SenFaseFlexjar() {
+  const [isApen, setIsApen] = useState<boolean>(true);
+  const [arenaBrukText, setArenaBrukText] = useState<string>();
+  const [andreTilbakemeldingerText, setAndreTilbakemeldingerText] =
+    useState<string>();
+  const [radioValue, setRadioValue] = useState<RadioOption | null>(null);
+  const sendFeedback = useFlexjarFeedback();
+  const { setStoredValue: setFeedbackDate } = useLocalStorageState<Date | null>(
+    StoreKey.FLEXJAR_SEN_FASE_FEEDBACK_DATE
+  );
+
+  function toggleApen() {
+    if (isApen) {
+      setRadioValue(null);
+      setAndreTilbakemeldingerText("");
+    }
+    setIsApen(!isApen);
+  }
+
+  function buildFeedbackString() {
+    const arenaBrukTextFeedback =
+      radioValue === RadioOption.JA
+        ? `[Hvordan bruker du Arena til dette: ${arenaBrukText}], `
+        : "";
+    return `${arenaBrukTextFeedback}[Har du noen tilbakemeldinger om denne siden: ${andreTilbakemeldingerText}]`;
+  }
+
+  const handleSubmit = () => {
+    if (radioValue) {
+      const feedbackString = buildFeedbackString();
+      const body: FlexjarFeedbackDTO = {
+        feedbackId: "SenFase",
+        feedback: feedbackString,
+        svar: radioValue,
+        app: "syfomodiaperson",
+      };
+      sendFeedback.mutate(body, {
+        onSuccess: () => {
+          setFeedbackDate(new Date());
+        },
+      });
+    }
+  };
+
+  return (
+    <div className="flex flex-col sticky bottom-0 self-end items-end z-[100]">
```

Comment:
Dobbeltsjekk gjerne at elementet ikke "flyter" over andre ting. Ser vi har denne feilen i dette repoet (på arbeidsuførhet-siden), men vet at vi fikset det i oversikten fordi da kunne ikke folk klikke på rader i tabellen. 
<img width="1157" alt="image" src="https://github.com/user-attachments/assets/b484046d-40a8-4c83-a309-8a58426cb5a2">

Blir ekstra dumt når den er ekspandert: 
<img width="1163" alt="image" src="https://github.com/user-attachments/assets/d2c4a5b8-afe4-4dfa-b8a1-2fc685e99e3a">

Reply:
Godt tenkt! Akkurat nå flyter den over andre ting, så jeg fikser det! Skal jeg lage en lapp på det for flexjaren på arbeidsuførhet? Er veldig enkelt å fikse

Reply:
Ja, det hørtes lurt ut med en egen lapp for den! Da minimerer vi også sjansen for å kopiere buggy kode inn i andre flexjarer også 👍🏼